.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseSetParser.cs

```diff
@@ -190,20 +223,143 @@ private DateTimeResolutionResult ParserTimeEveryday(string text, DateObject refD
             var ret = new DateTimeResolutionResult();
 
             var ers = this.config.TimeExtractor.Extract(text, refDate);
+            var ers1 = this.config.TimePeriodExtractor.Extract(text, refDate);
+
+            if (ers.Count == 1)
+            {
+                var afterStr = text.Replace(ers[0].Text, string.Empty);
+                var match = this.config.EachDayRegex.Match(afterStr);
+
+                if (match.Success)
+                {
+                    var pr = this.config.TimeParser.Parse(ers[0], DateObject.Now);
```

Comment:
Why passing "now" and not the existing reference time?

Reply:
replaced DateObject.Now with refdate.